controller:
  position_controller:
    natural_frequency: 1.
    damping_ratio: 1. # 理論的に振動しないギリギリの値
  rotation_controller:
    prediction_horizon: 1.
    prediction_steps: 10
    input_steps: 10
    weight_scale: 1.0e-6
    decay:
      rotation:
        roll: 0.2
        pitch: 0.2
        yaw: 0.2
      angular_velocity:
        x: 0.
        y: 0.
        z: 0.
    state_weight:
      rotation:
        roll: 100.
        pitch: 100.
        yaw: 100.
      angular_velocity:
        x: 1.
        y: 1.
        z: 1.
    input_weight: 1.0e-3
    input_rate_weight: 1.0e-3
